pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -27,6 +27,9 @@
 
 from pandas.tseries.offsets import BDay
 
+# We pass through a TypeError raised by numpy
```

Comment:
nit. you used iloc_msg in the other function (non-private); i would make these more verbose

slice_exception_msg